app/src/org/odk/collect/android/views/media/MediaLayout.java

```diff
@@ -204,39 +203,26 @@ else if (qrCodeContent != null ) {
             try {
                 int[] maxBounds = getMaxCenterViewBounds();
 
-                //If we didn't get an image yet, try for a norm
                 final String imageFilename = ReferenceManager._().DeriveReference(imageURI).getLocalURI();
                 final File imageFile = new File(imageFilename);
                 if (imageFile.exists()) {
+
                     Bitmap b = null;
                     try {
-                        Display display =
-                                ((WindowManager) getContext().getSystemService(Context.WINDOW_SERVICE))
-                                        .getDefaultDisplay();
-
-
-                        int screenWidth = display.getWidth();
-                        int screenHeight = display.getHeight();
-                        b =
-                            FileUtils
-                                    .getBitmapScaledToDisplay(imageFile, screenHeight, screenWidth);
+                        b = MediaUtil.inflateDisplayImage(getContext(), imageURI, maxBounds[0],
+                                maxBounds[1]);
                     } catch (OutOfMemoryError e) {
```

Comment:
this isn't possible anymore, right? Since the OOM is now caught inside of the Utility? Would clean up the code a bit to get rid of it.